readme.md

```diff
@@ -52,6 +52,7 @@
 - [whisper-openvino](https://github.com/zhuzilin/whisper-openvino) - Whisper running on OpenVINO.
 - [whisper.tflite](https://github.com/usefulsensors/openai-whisper) - Whisper running on TensorFlow Lite.
 - [Whisper variants](https://huggingface.co/models?other=whisper) - Various Whisper variants on Hugging Faces.
+- [Whisper-AT](https://github.com/YuanGongND/whisper-at) - Unified Speech and Non-Speech Audio Events Recogntion.
```

Comment:
Description should not be title-case. I think it would also be more useful for the description to succinctly describe how it's better than plain Whisper.

Reply:
Thanks! Just fixed based on your comment.